qiskit/synthesis/qft/__init__.py

```diff
@@ -13,3 +13,4 @@
 """Module containing stabilizer QFT circuit synthesis."""
 
 from .qft_decompose_lnn import synth_qft_line
+from .qft_decompose_full import synth_qft_full
```

Comment:
This decomposition method does not appear in the synthesis library API docs. Perhaps you would like to add it?

Reply:
Please add it (twice!) to this init file: 
https://github.com/Qiskit/qiskit/blob/main/qiskit/synthesis/__init__.py

Reply:
Good catch! Done in [6feb4fc](https://github.com/Qiskit/qiskit/pull/11463/commits/6feb4fc027e461a17b4fab68776b18bbf13603f3).